lisa/tools/python.py

```diff
@@ -89,3 +94,74 @@ def exists_package(self, package_name: str) -> bool:
     def uninstall_package(self, package_name: str) -> bool:
         result = self.run(f"uninstall {package_name} -y", force_run=True, sudo=True)
         return result.exit_code == 0
+
+
+class PythonVenv(Tool):
+    @property
+    def command(self) -> str:
+        path = self.get_venv_path().joinpath(f"bin/{self._python.command}")
+        if self.node.os.is_posix:
+            return path.as_posix()
+        else:
+            return str(path)
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    @property
+    def dependencies(self) -> List[Type[Tool]]:
+        return [Python]
+
+    def _create_venv(self, venv_path: str) -> PurePath:
+        cmd_result = self._python.run(
+            f"-m venv {venv_path}", force_run=True, shell=True
+        )
+        assert_that(
+            cmd_result.exit_code, f"fail to create venv: {venv_path}"
+        ).is_equal_to(0)
+        self._venv_path = PurePath(venv_path)
```

Comment:
Not sure if it works for both Linux and Windows. Use node.get_pure_path to get right version purepath of the node.